app/locales/gj/overlap.json

```diff
@@ -0,0 +1,11 @@
+{
+  "overlap_title": "આંતરછેદ તપાસો",
+  "overlap_para_1": "લીલા વર્તુળો તમારા સ્થાનોના ક્લસ્ટરો છે, ઝૂમ કરવા માટે ક્લિક કરો. લાલ માર્કર્સ તમારા રેકોર્ડ કરેલા સ્થાનો છે.\n\n ચક્કરવાળા જાંબુડિયા વર્તુળોમાં બતાવ્યા પ્રમાણે, પુષ્ટિ થયેલ કેસોની જાહેર માહિતી ડાઉનલોડ કરવા માટે બટન દબાવો. ",
+  "show_overlap": "સાર્વજનિક માહિતી તપાસો",
```

Comment:
Also, can you check this one
"show_overlap": "Click to view the public dataset",

Reply:
This seems alright. It says 'examine public information', which was the closest semantically to public dataset. I've clarified to include 'click'